assets/analytics.js

```diff
@@ -22,6 +30,15 @@
             trackingObject['type'] = 'PAGE_VIEW';
             track(trackingObject);
         },
+
+        addTrackingEnricher: function(trackingEnricher) {
+            enrichers.push(trackingEnricher);
+        },
+
         track: track
     };
+
+    loadedEvent = document.createEvent('Event');
+    loadedEvent.initEvent('holmesloaded', false, false);
+    window.dispatchEvent(loadedEvent);
```

Comment:
Does this work in all current browsers (incl. IE)?

Reply:
Yes. I tests this with a current FF, current Chrome and IE11.